iron-ajax.html

```diff
@@ -122,18 +122,46 @@
       },
 
       /**
-       * Optional raw body content to send when method === "POST".
+       * Body content to send with the request, typically used with "POST"
+       * requests.
        *
-       * Example:
+       * If body is a string it will be sent up raw with the request.
+       *
+       * If body is a plain object and no Content-Type is set, the object
+       * will be form encoded before sending.
+       *
+       * If Content-Type is set and iron-request knows how, the body will be
+       * encoded according to that Content-Type. Content-Types known:
+       *    * application/json
+       *    * application/x-www-form-urlencoded
```

Comment:
Perhaps provide a mapping of how each `Content-Type` will cause `body` to be encoded? We do this in other parts of the documentation: https://github.com/PolymerElements/iron-ajax/blob/master/iron-ajax.html#L148-L165